yhondanilo/chess_programas.rb

```diff
@@ -0,0 +1,43 @@
+
+ arr=Array.new(3){Array.new(3)}
+  arr[0][0]=1
+  for i in 0..3
+   for j in 0..3
+    if arr[i-1][j]==0
+     puts "true"
+     arr[i][j]=1
+    else
+     arr[i][j]=0
+    end
+    if arr[i][j-1] ==0
+     arr[i][j]=1
+    else
+     arr[i][j]=0
+    end
+   end
+  end
+
+
+
+
+arr=Array.new(2){Array.new(2)}
```

Comment:
Inconsistent indentation detected.<br>Surrounding space missing for operator '='.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.